CharacterCreation/03_Feats.txt

```diff
@@ -638,12 +638,20 @@ have no prerequisites.
 
 == Shotgun Expertise  ==
 
-0:	+10m to range R3	
-12:	-15% reload miss chance			
-24:	+5 Gun Reflex				
-36:	-0.5 miss chance @ all ranges			
-48:	+10% perception checks for awareness within 25m.			
-60:	+10% Base damage. If you hit armor with a higher APL than your shotgun, you deal half damage to its AS instead of no damage (and full damage to health if you deplete the AS).
+0:	+5m to range R1. If this would cause R1 to be greater than R2, 
+	also set R2 equal to the new R1.
+12:	May use Weapons - Shotgun in place of Acrobatics for checks to execute 
+	a physical maneuver following violent entry of a room.

```

Comment:
I think we should maybe formally write out how room-clearing works. Your bonus here has some real potential, but the mechanics it's resting on aren't developed enough to support it.

Reply:
For instance, the weapon's reflex save is supposed to be what counts in room clearing, not really Acrobatics.

Reply:
I was wary of weapon reflex at the time. I was specifically trying to describe a very active choice of Action to make it more skill-like and less reaction-like. I may not have succeeded in that, looking again.

Classifying as New Issue. Would you like to make it or shall I?